### DISCUSSION TITLE
which Sentinel-2 products are supported

### DISCUSSION THREAD

**Discussion by roybenhayun** [torchgeo/torchgeo#1568] [Q&A]
Hi,
when working with Sentinel-2 images, we are either using the L1C product or using Sen2Cor to build the L2A product (includes some atmospheric correction AFAIK).

is there any related difference to either L1C or L2A when using the [Sentinel2 dataset](https://torchgeo.readthedocs.io/en/stable/api/datasets.html#torchgeo.datasets.Sentinel2)?

**Comment by adamjstewart** [comment]
Our Sentinel-2 dataset should support both (let me know if this is not the case).
